src/rcc.rs

```diff
@@ -164,231 +166,117 @@ impl HSEClock {
 
 const HSI: u32 = 16_000_000; // Hz
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum PLLP {
+    Pllp2 = 0b00,
+    Pllp4 = 0b01,
+    Pllp6 = 0b10,
+    Pllp8 = 0b11,
```

Comment:
It's better to call them `Div2`, `Div4`, ...

Reply:
Done.